### PR TITLE
OADP 2164 rephrase

### PR DESCRIPTION
**Pull request by CarmiWisemon**
OADP - 1.1.6
OADP - 1.2.1

OCP - 4.11+

Resolves - https://issues.redhat.com/browse/OADP-2164

Preview - https://63978--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-features-plugins

### CODE REVIEW

File: modules/oadp-features.adoc
Comment (CarmiWisemon) [inline modules/oadp-features.adoc:1]: (Line 12) 
"Backup::
You can use OADP to back up all applications on the OpenShift Platform, or you can filter the resources by type, namespace, or label."

Does this also require to be rephrased?

Comment (stillalearner) [reply modules/oadp-features.adoc:1]: Backup looks good.

**Review by stillalearner — APPROVED** [review]
lgtm

File: modules/oadp-features.adoc [hunk 1]
```diff
@@ -21,7 +21,7 @@ You must exclude Operators from the backup of an application for backup and rest
 
 
 Restore::
-You can restore resources and PVs from a backup. You can restore all objects in a backup or filter the restored objects by namespace, PV, or label.
+You can restore resources and PVs from a backup. You can restore all objects in a backup or filter the objects to be restored by namespace, PV, or label.
```
Comment (dfitzmau) [inline modules/oadp-features.adoc:24]: Do we need the "to be restored" passive construct? I think "You can restore" covers the restoration operation. 

@mburke5678 . WDYT?

Comment (CarmiWisemon) [reply modules/oadp-features.adoc:24]: You can restore resources and PVs from a backup. You can restore all objects in a backup or filter the objects by namespace, PV, or label.

@stillalearner Is this phrasing OK with you?

Comment (CarmiWisemon) [reply modules/oadp-features.adoc]: @dfitzmau 
Thank you. I updated it based on your suggestion.

Comment (stillalearner) [reply modules/oadp-features.adoc]: @CarmiWisemon Looks good.

**Pull request merged**